system-tests/tests/load/cre/workflow-don-load-test-streams-crib.toml

```diff
@@ -0,0 +1,138 @@
+[jd]
+image = "job-distributor:0.9.0" #Currently this is hardcoded, leave this as it is
+
+# ----- MOCK CAPABILITIES -----
+# Trigger capability that simulates streams of data
+[[mock_capabilities]]
+name="streams-trigger"
+version= "2.0.0"
+type="trigger"
+description="streams trigger mock"
+
+# Target capability that simulates writing data to Ethereum
+[[mock_capabilities]]
+name="write_ethereum"
+version= "1.0.0"
+type="target"
+description="write trigger mock"
+
+# ----- TEST CONFIGURATION -----
+# Parameters for the Workflow DON load test
+[workflow_load]
+streams = 1000              # Number of streams to simulate
+jobs = 2                    # Number of jobs per stream
+
+# ----- INFRASTRUCTURE SETUP -----
+# Configures AWS-based CRIB infrastructure
+[infra]
+type = "crib"
+
+# CRIB deployment configuration for AWS
+[infra.crib]
+namespace = "crib-workflow-don-load-test"
+folder_location = "../../../../../crib/deployments/cre"  # Path to CRIB deployment files
+provider = "aws"
+
+# Team information for resource allocation and billing
+[infra.crib.team_input]
+team="cre"
+product="workflow-don"
+component="workflow-don"
+cost_center="capabilities-df-migration-performance-tests"
+
+# ----- WORKFLOW NODES CONFIGURATION -----
+# Setup for a 17-node Workflow DON cluster (includes bootstrap node)
+[[nodesets]]
+nodes = 17
+override_mode = "all"           # Apply same configuration to all nodes
+http_port_range_start = 10100   # Starting port for HTTP endpoints
+name = "workflow"
+
+# Database configuration for the workflow nodes
+[nodesets.db]
+image = "postgres:12.0"
+port = 13000
+volume_name = "workflow_volume"
+
+# Chainlink node configuration for workflow nodes
+[[nodesets.node_specs]]
+[nodesets.node_specs.node]
+image = "public.ecr.aws/chainlink/chainlink:2.23.0-ubuntu20" # Chainlink node image for AWS deployment
+user_config_overrides = """
+       [Feature]
+      LogPoller = true
+      FeedsManager = true
+      [OCR2]
+      Enabled = true
+      DatabaseTimeout = '1s'
+      [P2P.V2]
+      Enabled = true
+      ListenAddresses = ['0.0.0.0:5001']
+      """
+
+# ----- CAPABILITIES NODES CONFIGURATION -----
+# Setup for a 5-node Capabilities cluster  (includes bootstrap node)
+[[nodesets]]
+nodes = 5
+override_mode = "all"           # Apply same configuration to all nodes
+http_port_range_start = 10200   # Starting port for HTTP endpoints
+name = "capabilities"
+
+# Database configuration for the capabilities nodes
+[nodesets.db]
+image = "postgres:12.0"
+port = 13200
+
+# Chainlink node configuration for capabilities nodes
+[[nodesets.node_specs]]
+[nodesets.node_specs.node]
+image = "public.ecr.aws/chainlink/chainlink:2.23.0-ubuntu20" # Chainlink node image for AWS deployment
+user_config_overrides = """
+      [Feature]
+      LogPoller = true
+      FeedsManager = true
+      [OCR2]
+      Enabled = true
+      DatabaseTimeout = '1s'
+      [P2P.V2]
+      Enabled = true
+      ListenAddresses = ['0.0.0.0:5001']
+      """
```

Comment:
this can be deleted from all configs, because when we generate the config we add that bit anyway